### PR TITLE
Issue588 update overview essay

### LINKED ISSUE #588: update overview essay
The [extended essay](https://pages.nist.gov/chimad-phase-field/benchmarks/) includes language that requires maintenance any time a benchmark set is released. Suggested edits:
- [x] ~~provide links to codes mentioned by name~~
- [x] ~~provide NIST disclaimer if necessary~~
- [x] rephrase last paragraph, either describing the current state or (better) making a time-independent statement that links back to the updated list of benchmarks.

### PR DESCRIPTION
**Pull request by tkphd**
**Update Olle's Benchmarks Overview essay to reflect recent work.**

Closes #588.

These changes can be [viewed live][live-site].

**Remember to tear down the [live site][live-site] when merging
or closing this pull request.**

[live-site]: http://random-cat-980.surge.sh/benchmarks/

---
This change is [Reviewable](https://reviewable.io/reviews/usnistgov/pfhub/980)

### CODE REVIEW

File: benchmarks/index.md [hunk 1]
```diff
@@ -104,14 +104,35 @@ key features. The problems are nontrivial and exhibit differing
...
-equations for conserved and non-conserved order parameters. A second
-set has been tested on a workshop and Hackathon, and will be posted in
-the near future.
+The first set of Benchmark Problems, [BM1]({{ site.baseurl
+}}/benchmarks/benchmark1.ipynb/){:target="_blank"} and [BM2]({{
+site.baseurl }}/benchmarks/benchmark2.ipynb/){:target="_blank"},
+involve diffusion of a solute and grain growth. Technically, they use
+the Cahn-Hilliard equation for a conserved order parameter, and
+coupled Cahn-Hilliard and Allen-Cahn equations for conserved and
+non-conserved order parameters. The second set of Benchmark Problems,
```
Comment (amjokisaari) [inline benchmarks/index.md]: For each set of BM problems, might want to say what the intent was.  BM I:  to make sure your fundamentals are good.  BM II: length scales with dendritic growth, equilibrium linear elasticity.  BM III: extending to additional physics that are less commonly used for phase field.  BM IV: you said it already.

File: benchmarks/index.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ codes, but were still designed to be not too computationally
 demanding. The National Institute of Standards and Technology has
 played a coordinating role in the development and management of these
 problems, and also hosts a
-[website for the Micromagnetic Standard Problems](http://www.ctcms.nist.gov/~rdm/mumag.org.html){:target="_blank"},
+[website for the Micromagnetic Standard Problems][mumag]{:target="_blank"},
```
Comment (wd15) [inline benchmarks/index.md]: This link appears to be no longer working on the test site. Did you add the mumag link?

File: benchmarks/index.md [hunk 3]
```diff
@@ -104,14 +104,35 @@ key features. The problems are nontrivial and exhibit differing
...
+to test a simple, targeted aspect of either the numerical implementation
 or the physics.
 
-The first set of Benchmark Problems involve diffusion of a solute and
-grain growth. Technically, they use the Cahn-Hilliard equation for a
-conserved order parameter, and coupled Cahn-Hilliard and Allen-Cahn
-equations for conserved and non-conserved order parameters. A second
-set has been tested on a workshop and Hackathon, and will be posted in
-the near future.
+The first set of Benchmark Problems, [BM1]({{ site.baseurl
```
[the remark this answers is not in the document]
Comment (tkphd) [reply benchmarks/index.md]: That's how Olle originally wrote it. I believe, yes, we're referring to PFHub Benchmark Problems as proper nouns. I could go either way, so chose to leave it alone.

File: benchmarks/index.md (hunk 1 above)
Comment (tkphd) [reply benchmarks/index.md]: I like it.

File: benchmarks/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (tkphd) [reply benchmarks/index.md]: ... however, "benchmark problems" is not treated as a proper noun anywhere else in the essay, even in "These phase field benchmark problems are designed to exhibit several key features." I'll downgrade further mentions to regular nouns.

File: benchmarks/index.md (hunk 2 above)
Comment (tkphd) [reply benchmarks/index.md]: I did, then removed it, then added again... Fixed.

**Pull request merged**